source/features/download-folder-button.tsx

```diff
@@ -39,7 +42,7 @@ void features.add({
 		pageDetect.isRepoTree
 	],
 	exclude: [
-		pageDetect.isRepoRoot // Already has an native download ZIP button
+		() => select.exists('get-repo-controller') // Already has an native download ZIP button
```

Comment:
Can we disable this for enterprise too, while we're making changes?

Reply:
Enterprise support should come to `download-directory` soon, there’s a PR open

Reply:
😮

Reply:
For reference, this is https://github.com/download-directory/download-directory.github.io/pull/29

Reply:
Interesting, I have yet to see an enterprise install that would be publicly accessible, even with a token. Normally they're behind a VPN or a third party login, e.g. Okta.

Reply:
@busches indeed, but I don't think that's a hard requirement.